data/mods/Magiclysm/Spells/magus.json

```diff
@@ -63,5 +63,107 @@
     "base_casting_time": 600,
     "energy_source": "MANA",
     "base_energy_cost": 350
+  },
+  {
+    "id": "magus_eldritch_blast",
+    "type": "SPELL",
+    "name": "Eldritch Blast",
+    "description": "A blast of concentrated magical power that obliterates a large area.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 150,
+    "base_energy_cost": 200,
+    "difficulty": 5,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_aoe": 1,
+    "max_aoe": 5,
+    "aoe_increment": 0.5,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 1
+  },
+  {
+    "id": "magus_mana_bolt",
+    "type": "SPELL",
+    "name": "Mana Bolt",
+    "description": "A bolt of magical power that only damages your foes.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 100,
+    "base_energy_cost": 150,
+    "difficulty": 3,
+    "max_level": 20,
+    "min_damage": 10,
+    "max_damage": 50,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_range": 5,
+    "max_range": 30,
+    "range_increment": 1.5
+  },
+  {
+    "id": "magus_haste",
+    "type": "SPELL",
+    "name": "Haste",
+    "description": "This spell gives you an enormous boost of speed lasting a short period of time.",
+    "valid_targets": [ "self" ],
+    "effect": "target_attack",
+    "effect_str": "haste",
+    "effected_body_parts": [ "TORSO" ],
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "difficulty": 4,
+    "base_casting_time": 200,
+    "base_energy_cost": 300,
+    "max_level": 20,
+    "min_duration": 600,
+    "max_duration": 12000,
+    "duration_increment": 600
+  },
+  {
+    "id": "magus_mana_beam",
+    "type": "SPELL",
+    "name": "Mana Beam",
+    "description": "A beam of focused magical power that damages any foes in its path.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "line_attack",
+    "spell_class": "MAGUS",
+    "energy_source": "MANA",
+    "base_casting_time": 150,
+    "base_energy_cost": 200,
+    "difficulty": 4,
+    "max_level": 20,
+    "min_damage": 15,
+    "max_damage": 60,
+    "damage_increment": 3,
+    "damage_type": "none",
+    "min_range": 8,
+    "max_range": 25,
+    "range_increment": 1
+  },
+  {
+    "id": "magus_escape",
```

Comment:
this needs a "range" to work properly

Reply:
Ah, this is the spell I altered to try as a targeted attack and then edited back, I should have started from scratch.
Before I correct it...
What exact role does aoe play in teleporting?  the doc just says 'range spaces with aoe variation'.

Reply:
you teleport `range() +- aoe()` spaces

Reply:
OHHHH.. that's why I would get strange results and phase door further than I was expecting to based on what it said in the casting screen. Hmmm, well the idea was to basically get out of dodge with this, so what do you suggest?  higher range with less variation seems like the way to go with me.

Reply:
maybe range 25-40 with aoe 5-8?

Reply:
Wow, that's a lot higher and more generous than I was thinking, but that sounds good. When you wanna GTFO, you should GTFO. Which was the original name, should I change it back? ;)